docs/prometheus-integration.md

```diff
@@ -152,13 +152,13 @@ a configuration parameter for the Agent.
 in the conntrack table. This metric gets updated at an interval specified
```

Comment:
@jianjuns you can't modify this part of the file unless you modify the metrics definition in Go code. This part of the file is auto-generated.

Reply:
Ok. I updated Go code too.